Week-4/session-11/oopsEs6/ abstractionAndEncpasulation.js

```diff
@@ -0,0 +1,59 @@
+class BankAccount {
+  #accountNumber;
+  #balance;
+  #accountHolderName;
+
+  constructor(accountNumber, balance, accountHolderName) {
+    this.#accountNumber = accountNumber;
+    this.#balance = balance;
+    this.#accountHolderName = accountHolderName;
+  }
+
+  deposit(amount) {
+    this.#balance += amount;
```

Comment:
Consider adding validation for the amount parameter in the deposit() and withdraw() methods to ensure it's a positive number. This prevents accidental incorrect usage of these methods.